bitnami/concourse/templates/web/rbac.yaml

```diff
@@ -1,43 +1,32 @@
-{{- if .Values.web.enabled }}
-{{- if .Values.web.rbac.create -}}
----
+{{- if and .Values.web.enabled .Values.web.rbac.create -}}
 apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
-kind: ClusterRole
+kind: Role
 metadata:
   name: {{ template "concourse.web.fullname" . }}
   namespace: {{ .Release.Namespace | quote }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
-    app: concourse
+    app.kubernetes.io/component: web
     {{- if .Values.commonLabels }}
     {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
     {{- end }}
   {{- if .Values.commonAnnotations }}
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 rules:
-  - apiGroups: [""]
-    resources: ["secrets"]
-    verbs: ["get"]
-{{- end }}
-
----
-{{- $pspAvailable := (semverCompare "<1.25-0" (include "common.capabilities.kubeVersion" .)) -}}
-{{- if and $pspAvailable .Values.web.psp.create }}
-apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
-kind: ClusterRole
-metadata:
-  name: {{ template "concourse.web.fullname" . }}
-  labels:
-    app.kubernetes.io/component: web
-    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
-    heritage: {{ .Release.Service }}
-    release: {{ .Release.Name }}
-rules:
+  {{- $pspAvailable := (semverCompare "<1.25-0" (include "common.capabilities.kubeVersion" .)) -}}
+  {{- if and $pspAvailable .Values.web.psp.create }}
   - apiGroups: ['policy']
     resources: ['podsecuritypolicies']
     verbs:     ['use']
     resourceNames:
       - {{ template "concourse.web.fullname" . }}
+  {{- end }}
+  - apiGroups: [""]
+    resources: ["secrets"]
```

Comment:
I see that before it was a ClusterRole, could you confirm that it doesn't need secrets from all the namespaces? I'm not sure what the secret is for inside the Concourse software.

Reply:
TBD (expect updates on this)